pkg/network/ebpf/c/protocols/http2/defs.h

```diff
@@ -14,6 +14,9 @@
 #define HTTP2_FRAME_HEADER_SIZE 9
 #define HTTP2_SETTINGS_SIZE 6
 
+// According to https://datatracker.ietf.org/doc/html/rfc7540#section-6.4, the frame includes a 32-bit error code integer.
+#define HTTP2_RST_ERROR_SIZE 32
```

Comment:
byte we're reading bytes
it is better to remove this define and use `sizeof(<var name>)`